bot.env

```diff
@@ -3,3 +3,7 @@ DB_HOST="mysql_db"
 DB_PORT=3306
 DB_USER="root"
 DB_PASSWORD="pass"
+REDIS_HOST="cache"
+REDIS_PORT=6379
```

Comment:
Its better if these secrets are encoded. We can create store them as secrets in kubernetes dev environment